org.adempiere.ui/src/org/compiere/minigrid/ColumnInfo.java

```diff
@@ -117,6 +117,27 @@ public ColumnInfo (String colHeader, String colSQL, Class<?> colClass,
 		setKeyPairColSQL(keyPairColSQL);
 	}   //  ColumnInfo
 
+	/**
+	 *  Create Info Column
+	 *
+	 *  @param colHeader Column Header
+	 *  @param colSQL    SQL select code for column
+	 *  @param colClass  class of column - determines display
+	 *  @param readOnly  column is read only
+	 *  @param columnName Column Name
+	 */
+	public ColumnInfo (String colHeader, String colSQL, Class<?> colClass, boolean readOnly, String columnName)
+	{
+		setColHeader(colHeader);
+		setColSQL(colSQL);
+		setColClass(colClass);
+		setReadOnly(readOnly);
+		setColumnName(columnName);
+	}   //  ColumnInfo	

```

Comment:
where possible, our usual style is to redirect this to existing constructor. for this case, I think it should be just { this(colHeader, colSQL, colClass, readOnly, false, null, columnName); }

Reply:
done